app/controllers/works_controller.rb

```diff
@@ -18,61 +18,66 @@ class WorksController < ApplicationController
   before_filter :set_instance_variables, only: [:new, :create, :edit, :update, :manage_chapters, :preview, :show, :navigate, :import]
   before_filter :set_instance_variables_tags, only: [:edit_tags, :update_tags, :preview_tags]
 
-  before_filter :clean_work_search_params, only: [:search, :index, :collected]
-
   cache_sweeper :collection_sweeper
   cache_sweeper :feed_sweeper
 
   # we want to extract the countable params from work_search and move them into their fields
   def clean_work_search_params
-    if params[:work_search].present? && params[:work_search][:query].present?
+    #https://aaronlasseigne.com/2014/07/20/know-ruby-clone-and-dup/
```

Comment:
Drive-by comment! This page is informative, but I'm not sure which part(s) of it is relevant to the code. A more helpful comment might be something like "We use dup instead of clone here because we don't want to copy the frozen state of the object" or "Using dup here means the original object also gets modified when we do the gt/lt swap below" or whatever.